### ISSUE TITLE
Problem with synchronization across multiple devices

### ISSUE THREAD

**Issue by qoheniac** [mtotschnig/MyExpenses#586]
If I add my webdav location to the app on a second device I get the following message:

> 1 accounts could not be configured to be synced with the backend. An account with the same uuid already exists on the remote backend. The conflict can be resolved manually from the entry's context menu.

The context menu only contains the option to remove the entry. So how can I resolve this problem and what is causing this? Some maybe relevant information: I use Nextcloud, my domain contains an umlaut and the smartphones are both Fairphone 2 with MicroG and the app installed from F-Droid.

By the way, the message shows up on the bottom of the screen only very shortly and I had to make a screenshot to read it. It also did not show the complete text but instead ended in … and I found the rest in the source code. After that it seems to work, while it does not. It even shows these sync messages, saying there were no changes to be synced. Nothing pointing to the fact anymore that synchronization isn't working. Maybe that's an issue in itself.

**Comment by mtotschnig** [comment]
With the current implementation, it is not possible to link identical existing accounts on two devices. Probably those have been created by copying a backup from one device to the other.  Is this your case? I agree that this should be possible, but currently it is not, because when you restore a backup, sync accounts are not restored.
The "entry's context menu" refers not to the sync backend, but to the accounts, that are listed as their children. For an account that is not linked, but exists both locally and remote, the context menu has the command "Start synchronizsation", which then opens a dialogue which allows you to decide between copying local data to remote thus erasing remote data or copying remote data to local thus erasing local data.
Thanks for reporting the usability problem of the message not being readable. I will try to remove this for the next version.

**Comment by qoheniac** [comment]
I deleted the backend from both devices and then the local data from device 2 plus the remote data from the server. I then added the backend on device 1 again and synced the data to the server and after that I also added the backend on device 2 again. Now everything is synced and there were no error messages. I hope this solved it. I'm still not sure how I got into this as I never copied any local data manually from one device to the other. I will comment again if the problem reappears, but for the moment I guess this issue is solved.
